src/main/resources/META-INF/mods.toml

```
@@ -3,18 +3,24 @@ loaderVersion = "[40,)"
 issueTrackerURL = "https://github.com/refinedmods/refinedstorage"
 license="MIT"
 [[mods]]
-modId="refinedstorage"
-version="${version}"
-displayName="Refined Storage"
+modId = "refinedstorage"
+version = "123"
+displayName = "Refined Storage"
 displayURL="https://refinedmods.com/refined-storage"
 logoFile="logo.png"
 authors = "Refined Mods"
 description = '''
 An elegant solution to your hoarding problem
 '''
 [[dependencies.refinedstorage]]
-modId="forge"
-mandatory=true
-versionRange="[40.1,)"
-ordering="NONE"
-side="BOTH"
\ No newline at end of file
+modId = "forge"
+mandatory = true
+versionRange = "[40.1,)"
+ordering = "NONE"
+side = "BOTH"
+[[dependencies.refinedstorage]]
+modId = "jei"
+mandatory = true
+versionRange = "[9.7.1,)"
+ordering = "NONE"
```

I think you can make it so that JEI is not mandatory, but if it is present then it needs to be in the specific version range. If it is not mandatory, n ordering must be specified, but the specific ordering doesn't matter here so I just picked one.

```suggestion
modId = "jei"
mandatory = false
versionRange = "[9.7.1,)"
ordering = "BEFORE"
```